index.html

```diff
@@ -231,5 +231,22 @@ <h3>LET'S CONNECT</h3>
               </ul>
              <div class="divider"></div>
        </section>
+       <footer class="contact-form">
+        <form class="contact-info" action="https://formspree.io/f/mwkyenjw" method="POST">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <div class="input-container">
```

Comment:
- [ ] [OPTIONAL]: I want to point out that your structure is not incorrect, but it could be better. For instance, you could have the `<h2>` and `<p>` outside the form, but inside the footer section, this way the `<div class="input-container">` wouldn't be needed, you could simply add that class to the `<form>`.